src/main/webapp/WEB-INF/tags/instructor/home/coursePanel.tag

```diff
@@ -15,9 +16,30 @@
             <div class="mobile-margin-top-10px col-sm-6">
                 <span class="mobile-no-pull pull-right">
                     <c:forEach items="${courseTable.buttons}" var="button">
-                        <a data-toggle="tooltip" data-placement="top" ${button.attributesToString}>
-                            ${button.content}
-                        </a>
+                        <c:choose>
+                            <c:when test="${fn:length(button.nestedElements) gt 0}">
+                                <div class="dropdown courses-table-dropdown">
+                                    <button class="btn btn-primary btn-xs dropdown-toggle" type="button" data-toggle="dropdown">
```

Comment:
You're right, it makes less sense in this case, I was just thinking about that because of @damithc's comment on the UI standardization - after your explanation it seems reasonable to leave it as it is, any comments from @damithc?